Update register-server.adoc

PabloCamino opened the pull request:
Rearranged content and added a link to an example JVM documentation on CACerts store

PabloCamino committed:
Update register-server.adoc

Rearranged content and added a link to an example JVM documentation on CACerts store

Commit:
edits to PR

IsaacEldridge reviewed (dismissed):
@PabloCamino Can you check my edits?

PabloCamino committed:
Update register-server.adoc

Changed some words and used neutral vendor reference as not all JVMs are provided by Oracle, though all should follow its standards.

PabloCamino committed:
Changed tmp cer file name

PabloCamino reviewed (commented):
I have made some changes for clarification

Commit:
edit

Commit:
edit

kbostrom commented on modules/ROOT/pages/register-server.adoc:
```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
 endif::[]
 
-If you encounter an error when performing the steps in xref:runtime-manager::servers-create.adoc[Create a Server in Runtime Manager]:
+If you encounter a Mule runtime engine registration error when xref:runtime-manager::servers-create.adoc[creating a server in Runtime Manager], you may need to configure Mule to trust the TLS certificates you are using. 
```
may > might, per style ref.

IsaacEldridge replied:
got it

kbostrom commented on modules/ROOT/pages/register-server.adoc:
```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
+
+. Get the public certificate for Anypoint Platform Private Cloud Edition.
+. Add the certificate to the truststore of the JVM you are using for the Mule runtime engine.
+
```
I also want to say that "you are using" (twice above) should be "you use," but don't feel particularly strongly about that....

kbostrom commented on modules/ROOT/pages/register-server.adoc:
```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
+. Get the public certificate for Anypoint Platform Private Cloud Edition.
+. Add the certificate to the truststore of the JVM you are using for the Mule runtime engine.
+
+Using the following example, set `$ANYPOINT_DNS` to your DNS for Anypoint Platform Private Cloud Edition. Replace the keystore location and password for your JVM's truststore.
```
JVM need not be possessive (as an adjective rather than a noun)

kbostrom commented on modules/ROOT/pages/register-server.adoc:
```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
-In the previous example, $ANYPOINT_DNS should be set to your DNS for Anypoint Platform Private Cloud Edition.
-Replace the location and password for your JVM's truststore.
+
+For more information about the CA cert truststore, refer to your JVM vendor's documentation. For example,  https://docs.oracle.com/en/java/javase/17/security/java-security-overview1.html#GUID-EB29B931-D1DE-420B-849C-D731A8DF9CDC[Oracle documentation^].
```
I ran a search and did not see `certificate` abbreviated to `cert` except in the context of `cacert`?

kbostrom replied:
```suggestion
For more information about the CA certificate truststore, refer to your JVM vendor documentation: for example,  https://docs.oracle.com/en/java/javase/17/security/java-security-overview1.html#GUID-EB29B931-D1DE-420B-849C-D731A8DF9CDC[Oracle documentation^].
```

kbostrom reviewed (dismissed):
PR

Commit:
post review edits

kbostrom reviewed (approved):
PR

IsaacEldridge commented:
@PabloCamino I'm closing this PR and merging this content on https://github.com/mulesoft/docs-private-cloud/pull/242

I had unverified commits, as my GPG was not previously set up
